.github/workflows/app-distribute.yml

```diff
@@ -0,0 +1,39 @@
+name: App Distribute CI
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  build_virtualevent_sample_app:
+    name: Build and Distribute VirtualEvent Sample App
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 11
+        uses: actions/setup-java@v2
+        with:
+          distribution: adopt
+          java-version: 11
+      - name: Prepare environment
+        run: |
+          git fetch --unshallow
```

Comment:
why `git fetch --unshallow`?

Reply:
That's how we have it in our sample app. Not sure what was the reason for doing so.

Reply:
I would try to remove and test it or ask the author why putting it there... it doesn't seem to be adding anything to me.